modules/kobalt-plugin-api/src/main/kotlin/com/beust/kobalt/api/Project.kt

```diff
@@ -15,18 +19,20 @@ open class Project(
         @Directive open var artifactId: String? = null,
         @Directive open var packaging: String? = null,
         @Directive open var dependencies: Dependencies? = null,
-        @Directive open var sourceSuffix : String = "",
-        @Directive open var description : String = "",
-        @Directive open var scm : Scm? = null,
```

Comment:
FYI, I try to put cosmetic changes (reformatting, reindenting, etc...) in separate PR so they don't obscure the main PR. No big deal for now.